### PR TITLE
Team member #2 : Create Class and Properties.

### PR DESCRIPTION
**Pull request by jorgegoco**
- Created MusicAlbum class in a separate .rb file.
- Created Genre class with an association to the Item class (in a separate .rb file).

- All MusicAlbum class properties visible in the diagram are defined and set up in the constructor method.

- All Genre class properties visible in the diagram are defined and set up in the constructor method.

### CODE REVIEW

File: spec/genre_spec.rb [hunk 1]
```diff
@@ -9,12 +9,12 @@
       expect(genre).to have_attributes(name: 'Abc')
       expect(genre).to have_attributes(items: [])
     end
-    it 'add an item to a genre (add_item method)', positive: true do
-      genre = Genre.new name: 'Abc'
+    # it 'add an item to a genre (add_item method)', positive: true do
+    #   genre = Genre.new name: 'Abc'
```
Comment (akos29) [inline spec/genre_spec.rb:13]: [OPTIONAL]: Please remove codes you are not applying to have a clean code. You can always find it from the GitHub version controlling 😊

File: spec/genre_spec.rb [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
 
-      genre.add_item item: 'The Artist'
-      expect(genre).to have_attributes(items: ['The Artist'])
-    end
+    #   genre.add_item item: 'The Artist'
+    #   expect(genre).to have_attributes(items: ['The Artist'])
+    # end
```
Comment (akos29) [inline spec/genre_spec.rb:17]: Kindly remove the comments.

**Review by akos29 — APPROVED** [review]
STATUS: APPROVED 🟢
Hi @jorgegoco 👋,
Your project is complete! There is nothing else to say other than... it's time to close it :shipit:
Congratulations! 🎉
VeryNiceNiceGIF

File: spec/genre_spec.rb (hunk 2 above)
Comment (jorgegoco) [reply spec/genre_spec.rb:17]: Ok, thanks for the suggestion, and for the approval.

**Pull request merged**